src/extensions/service-managers/van-api-keys-manager/index.js

```diff
@@ -0,0 +1,68 @@
+/// All functions are OPTIONAL EXCEPT metadata() and const name=.
+/// DO NOT IMPLEMENT ANYTHING YOU WILL NOT USE -- the existence of a function adds behavior/UI (sometimes costly)
+
+import { getFeatures } from "../../../server/api/lib/config";
+import { cacheableData, r } from "../../../server/models";
+import { getConfig } from "../../../server/api/lib/config";
+import { getSecret, convertSecret } from "../../secret-manager";
+
+export const name = "van-api-keys-manager";
+
+export const metadata = () => ({
+  // set canSpendMoney=true, if this extension can lead to (additional) money being spent
+  // if it can, which operations below can trigger money being spent?
+  displayName: "VAN setup manager",
+  description: "Used for VAN integration without environment variables",
+  canSpendMoney: false,
+  moneySpendingOperations: ["onCampaignStart"],
+  supportsOrgConfig: true,
+  supportsCampaignConfig: false
+});
+
+export async function getOrganizationData({ organization, user, loaders }) {
+  // MUST NOT RETURN SECRETS!
+  return {
+    // data is any JSON-able data that you want to send.
+    // This can/should map to the return value if you implement onOrganizationUpdateSignal()
+    // which will then get updated data in the Settings component on-save
+    data: {},
```

Comment:
you want to pass data from previous settings.  The tricky part here is you don't want to send secrets out in plaintext, so maybe send a variable of e.g. `ngpVanApiKey_isSet` or something.